op-batcher/batcher/espresso.go

```diff
@@ -132,7 +132,7 @@ func (l *BatchSubmitter) espressoBatchLoadingLoop(ctx context.Context, wg *sync.
 		l.RollupConfig.L2ChainID.Uint64(),
 		l.L1Client,
 		l.Espresso,
-		l.EspressoLightClient,
+		nil, // TODO (Keyao) BatchSubmitter doesn't have field EspressoLightClient.
```

Comment:
Note this TODO. We need to either remove this field or fix the `nil` depending on whether to use the light client.

Reply:
Follow-up: https://app.asana.com/1/1208976916964769/project/1209392461754458/task/1209980878478924?focus=true